meetings/2019-03-f2f.md

```diff
@@ -0,0 +1,247 @@
+* Joe Alicata @wirelessjoe
+* Levi Durfee @levidurfee
+* Léonie Watson @ljwatson
+
+### What is the TSC
+
+* Technical Steering Committee.
+* Inspired by NodeJS style terminology.
+* Responsible for handling project day to day activity.
+* Roadmap, direction. Project management / tech lead distributed throughout a group.
+* TSC delegate to working groups.
+* 3 out of 7 members are from Google for now.
+* Goal is that no more than 1/3 of members are from the same company.
+
+### Scope and Role of the AC
+
+* AC is Advisory to the TSC.
+* Advice is non-binding.
+* AC can provide opinions / advice / findings.
+* Once we have a Foundation, we will have a board
+  * AC should be represented on the Board
+  * TSC should be represented on the Board
+  * The Board will be respnsible for finance, IP, and governance
+* AC should advise on Foundation's formation
+  * Membership cost
+  * Diversity
+  * Policies
+* Should AMP be a "sub-group" of W3C?
+* W3C is an SDO (Standards Developing Organization), so probably not right.
+* But probably needs a channel to W3C / WHAT-WG
+  * If W3C/WHAT-WG work together, AMP should work with them (but not be a part of them)
+     * Liaison with SDOs is part of the AC's mandate.
+
+### Membership of AC
+
+* At the moment a mix of personal invitees, companies which have been working with AMP for a while, and people who signed up.
+* AMP Vision - https://www.ampproject.org/about/mission/
+  * Right intent but too vague
+* [issue meta#9][meta9]
+  * No - AMP AC needs "critical friends"
+  * We think that the vision / mission needs works
+  * Vision: A strong, user-first open web forever.
+  * **AI(@edent, @ljwatson): draft an AC opinion on the topic.**
+  * Once that opinion is drafted and published by the AC, and depending on the resolution of the TSC, the AC will revisit [issue meta#9][meta9]. The current sentiment is that those restrictions shouldn't apply as long as the AC's membership is properly balanced.
+* How do we get more members and more involvement from existing members?
+* Target size of AC is 6 - 12 member.
+* Current size is 16.
+* No fixed size.
+* AC must be representative of different stakeholders
+* Ideally, we'd love for AC members to be elected? In practice, that's impossible.
+* We've received requests from people to join
+* Membership length is 1 year.
+  * Members in good standing will be asked to rejoin.
+  * Members are under no obligation to rejoin
+  * Year starts on Jan, 1st.
+  * Members can resign at any time
+  * We need a concept of "Good Standing" (see for example [W3C's definition](https://www.w3.org/2005/10/Process-20051014/process.html#good-standing)), still TBD.
+* **AI(@tobie): open a pull request against the AC's work mode document to reflect agreed upon term durations.**
+* Current AC members approve any new AC member
+  * Consensus based process
+  * AC want to ensure a diverse group of representative (diversity of location, thought, industry, gender, disability, race, etc)
```

Comment:
AC wants to ensure a diverse group of representative (diversity of location, thought, industry, gender, disability, race, etc)